setup.py

```diff
@@ -49,8 +49,7 @@ def find_version(*file_paths):
 
 
 INSTALL_REQUIRES = [
-    "jinja2>=2.10.1, <2.11.0",
```

Comment:
Not sure why this version range crept in, I think we should be good to up the requirement.